worksheet_D/oxo.py

```diff
@@ -0,0 +1,131 @@
+class OxoBoard:
+
+    def __init__(self):
+        """Define board coordinates and set values to 0"""
+        self.oxoboard = {(0, 0): 0, (1, 0): 0, (2, 0): 0,
+                         (0, 1): 0, (1, 1): 0, (2, 1): 0,
+                         (0, 2): 0, (1, 2): 0, (2, 2): 0}
```

Comment:
OK, but initialising the dictionary with a loop woudl be neater (and more expandable e.g. to n*n boards)